massa-client/src/repl.rs

```diff
@@ -351,8 +351,41 @@ impl Output for Vec<IpAddr> {
 
 impl Output for Vec<OperationInfo> {
     fn pretty_print(&self) {
-        for operation_info in self {
-            println!("{}", operation_info);
+        for info in self {
+            print!("Operation {}", style(info.id).color256(175));
```

Comment:
it is possible to have the color in enum or constant ?

Reply:
For sure, I'll try to do something more generic to have something more elegant